pydantic/validators.py

```diff
@@ -214,6 +215,13 @@ def path_exists_validator(v) -> Path:
     return v
 
 
+def callable_validator(v) -> Callable:
+    if callable(v):
+        return v
```

Comment:
I think we will also need to support checking the arguments of a callable, but that's much more work let's start with this.

Please add a comment here (and in the docs) saying that we're just doing this simple check.